content/en/docs/components/image/_index.md

```diff
@@ -2,7 +2,7 @@
 title: Image reflector and automation controllers
 linkTitle: Image Automation Controllers
 description: "The GitOps Toolkit Image Automation Controllers documentation."
-weight: 5
+weight: 950
```

Comment:
Not sure what these changes are for...? AFAIK `weight` is only considered within the context of its directory. So you can just go `1`..`n` within a directory and don't need to worry about e.g. the weight of its section.

Reply:
It makes it easier if you don't have colliding weights

I wouldn't use 1-n , but try to use increments of 10 because it makes it easier to insert pages between

Larger set of numbers makes it easier to rearrange content

Reply:
But they don't collide... you can have `weight: 5` under `installation` and `components` and it will just be fine.